test/Microsoft.AspNet.Cors.Core.Test/CorsOptionsTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Xunit;
+
+namespace Microsoft.AspNet.Cors.Core.Test
+{
+    public class CorsOptionsTest
+    {
+        [Fact]
+        public void CookieName_SettingNullValue_Throws()
+        {
+            // Arrange
+            var options = new CorsOptions();
+
+            // Act & Assert
+            var ex = Assert.Throws<ArgumentNullException>(() => options.DefaultPolicyName = null);
+            Assert.Equal("The 'DefaultPolicyName' property of 'Microsoft.AspNet.Cors.Core.CorsOptions' must not be null." +
+                         Environment.NewLine + "Parameter name: value", ex.Message);
```

Comment:
Validate the message of an exception doesn't seem to be a stable tests, it may break easily when product is localized , the type name is changed or the format is updated.